cloverage/src/cloverage/coverage.clj

```diff
@@ -123,9 +123,15 @@
                        :regex-patterns regex-patterns})
   (let [namespaces (map name
                         (cond
-                          (and (empty? ns-paths) (empty? regex-patterns)) '()
-                          (empty? ns-paths) (ns-find/find-namespaces (cp/classpath))
-                          :else (ns-find/find-namespaces (map io/file ns-paths))))]
+                          (and (empty? ns-paths) (empty? regex-patterns))
+                          '()
+
+                          (empty? ns-paths)
+                          (distinct (ns-find/find-namespaces (filter #(.isDirectory ^java.io.File %)
+                                                                     (cp/system-classpath))))
```

Comment:
for me at least from the REPL with Java 9+ `cp/classpath` doesn't include everything by `cp/system-classpath` does.

Reply:
Yep! See https://github.com/latacora/graal-empty-classpath-reproducer for more examination about a way this specifically can fail (but generally we probably want system-classpath).